doc/python/subplots.md

```diff
@@ -37,9 +37,11 @@ jupyter:
 
 #### Simple Subplot
 
-Figures with subplots are created using the `make_subplots` function from the `plotly.subplots` module.
+Figures with subplots are created using the `make_subplots()` function from the `plotly.subplots` module. 
 
-Here is an example of creating a figure with two scatter traces in side-by-side subplots.
+This function accepts two integer parameters (`rows` and `cols`), which specifiy how the subplots should be laid out.
```

Comment:
this is probably misleading, as this function accepts many other parameters as well...